Tests/test_GenomeDiagram.py

```diff
@@ -213,14 +218,18 @@ def test_color_conversions(self):
         translator = ColorTranslator()
 
         # Does the translate method correctly convert the passed argument?
-        assert translator.float1_color((0.5, 0.5, 0.5)) == translator.translate((0.5, 0.5, 0.5)), \
-            "Did not correctly translate colour from floating point RGB tuple"
-        assert translator.int255_color((1, 75, 240)) == translator.translate((1, 75, 240)), \
-            "Did not correctly translate colour from integer RGB tuple"
-        assert translator.artemis_color(7) == translator.translate(7), \
-            "Did not correctly translate colour from Artemis colour scheme"
-        assert translator.scheme_color(2) == translator.translate(2), \
-            "Did not correctly translate colour from user-defined colour scheme"
+        assert translator.float1_color((0.5, 0.5, 0.5)) == translator.translate(
+            (0.5, 0.5, 0.5)
+        ), "Did not correctly translate colour from floating point RGB tuple"
+        assert translator.int255_color((1, 75, 240)) == translator.translate(
+            (1, 75, 240)
+        ), "Did not correctly translate colour from integer RGB tuple"
+        assert translator.artemis_color(7) == translator.translate(
+            7
+        ), "Did not correctly translate colour from Artemis colour scheme"
+        assert translator.scheme_color(2) == translator.translate(
+            2
+        ), "Did not correctly translate colour from user-defined colour scheme"
```

Comment:
These assert statements look ugly - they should be using ``self.assertEqual(...)`` instead which will look nicer.

Reply:
Done on #2767 , thank you, I'll close this one.